src/ReviewPage.js

```diff
@@ -1,17 +1,81 @@
 import React, { useState } from "react";
 import "./CSS/reviewpage.css";
-import { MDBInput } from "mdb-react-ui-kit";
+import { MDBInput, MDBScrollspyNavLink } from "mdb-react-ui-kit";
 import axios from "axios";
+import UserPage from "./UserPage";
+import AlbumTable from "./Tables/AlbumTable";
+import {
+  useNavigate,
+  Navigate,
+} from "react-router-dom";
 
 function ReviewPage(props){
+
+  const [user, setUser] = useState({ albums: []
+
+  });
+
+  const [album, setAlbum] = useState({ album_name: ""
+
+  });
+
+  const [reviewData, setReview] = useState({ 
+    owner: props.userData.username,  
+    content: "" , 
+    rating: "",
+    reviewedItem: ""});
+
+  let navigate = useNavigate();
+
   var URL = 'http://localhost:5000';
   // var URL = 'https://musiccatalogbe.herokuapp.com';
 
   if (process.env.NODE_ENV === "production"){
     URL = 'https://musiccatalogbe.herokuapp.com';
   }
 
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    if (name === "album_name") setAlbum({ ...album, album_name: value });
+    //else if (name === "album") setReview({ ...reviewData, album: value });
+    else if (name === "review") setReview({ ...reviewData, review: value });
+    else if (name === "rating") setReview({ ...reviewData, rating: value });
+  }
+
+  function removeAlbum(index) {
+    const updated = user.albums.filter((character, i) => {
```

Comment:
this is a backwards way to remove things (adding all good ones and not the deleting item)- is there no way to edit and filter the same collection?

Reply:
From what i know, there is no delete function so a removal of an object from a list must be done this way

Reply:
^^